bin/GameIndex.dbf

```diff
@@ -34767,6 +34791,14 @@ vuClampMode = 3 // fix white shiny weapons
 Serial = SLPS-29005
 Name   = Xenosaga Episode 1 - Der Wille zur Macht [Reloaded]
 Region = NTSC-J
+[patches = A3D63039]
+	// Reduces JPEG quality to its minimum for save file thumbnails. This prevents
+	// the game from calling its own exit function, which was strangely its safety
+	// mechanism for when a captured thumbnail exceeded 4 KB in size. 
+	comment=Save Point Crash Prevention
+	author=pandubz, PSI, turtleli
+	patch=1,EE,20285f48,extended,24040000
+[/patches]
```

Comment:
I think this patch should be removed. Reloaded is different from the other NTSC-J versions.